cobra/core/model.py

```diff
@@ -500,8 +500,18 @@ def add_boundary(self, metabolite, type="exchange", reaction_id=None,
         >>> demand.build_reaction_string()
         'atp_c --> '
         """
-        types = dict(exchange=("EX", -ub, ub), demand=("DM", 0, ub),
-                     sink=("SK", -ub, ub))
+        types = {"exchange": ("EX", -ub, ub),
```

Comment:
```suggestion
        types = {
            "exchange": ("EX", -ub, ub),
            "demand": ("DM", 0, ub),
            "sink": ("SK", -ub, ub)
        }
```

Reply:
Also, this function should make use of the bounds of the global configuration singleton rather than using these default arguments.